docs/assembly.rst

```diff
@@ -612,7 +612,7 @@ at position ``0x40`` in memory. If you want to allocate memory, just use the mem
 from that point on and update the pointer accordingly.
 
 The first 64 bytes of memory can be used as "scratch space" for short-term
-allocation. The 32 bytes after the free memory pointer (i.e. starting at ``0x60``)
+allocation. The 32 bytes after the free memory pointer (i.e. starting at ``0x80``)
```

Comment:
Actually, the 32 bytes after 0x60 are meant to be always zero and the text is correct in that case.

The "general purpose memory" starts at 0x80.

Perhaps this needs to be improved.